The token has not been saved to the git credentials helper. Pass `add_to_git_credential=True` in this function directly or `--add-to-git-credential` if using via `huggingface-cli` if you want to set the git credential as well.
Token is valid (permission: write).
Your token has been saved to C:\Users\gracz\.cache\huggingface\token
Login successful
The system cannot find the path specified.
[nltk_data] Downloading package wordnet to
[nltk_data]     C:\Users\gracz\AppData\Roaming\nltk_data...
[nltk_data]   Package wordnet is already up-to-date!
[nltk_data] Downloading package punkt to
[nltk_data]     C:\Users\gracz\AppData\Roaming\nltk_data...
[nltk_data]   Package punkt is already up-to-date!
[nltk_data] Downloading package omw-1.4 to
[nltk_data]     C:\Users\gracz\AppData\Roaming\nltk_data...
[nltk_data]   Package omw-1.4 is already up-to-date!
Requirement already satisfied: evaluate in c:\python312\lib\site-packages (0.4.2)
Requirement already satisfied: nltk in c:\python312\lib\site-packages (3.8.1)
Requirement already satisfied: rouge_score in c:\python312\lib\site-packages (0.1.2)
Collecting bert_score
  Using cached bert_score-0.3.13-py3-none-any.whl.metadata (15 kB)
Requirement already satisfied: datasets>=2.0.0 in c:\python312\lib\site-packages (from evaluate) (2.20.0)
Requirement already satisfied: numpy>=1.17 in c:\users\gracz\appdata\roaming\python\python312\site-packages (from evaluate) (1.26.4)
Requirement already satisfied: dill in c:\python312\lib\site-packages (from evaluate) (0.3.8)
Requirement already satisfied: pandas in c:\users\gracz\appdata\roaming\python\python312\site-packages (from evaluate) (2.2.0)
Requirement already satisfied: requests>=2.19.0 in c:\python312\lib\site-packages (from evaluate) (2.32.3)
Requirement already satisfied: tqdm>=4.62.1 in c:\python312\lib\site-packages (from evaluate) (4.66.4)
Requirement already satisfied: xxhash in c:\python312\lib\site-packages (from evaluate) (3.4.1)
Requirement already satisfied: multiprocess in c:\python312\lib\site-packages (from evaluate) (0.70.16)
Requirement already satisfied: fsspec>=2021.05.0 in c:\python312\lib\site-packages (from fsspec[http]>=2021.05.0->evaluate) (2024.2.0)
Requirement already satisfied: huggingface-hub>=0.7.0 in c:\python312\lib\site-packages (from evaluate) (0.23.0)
Requirement already satisfied: packaging in c:\python312\lib\site-packages (from evaluate) (23.2)
Requirement already satisfied: click in c:\python312\lib\site-packages (from nltk) (8.1.7)
Requirement already satisfied: joblib in c:\python312\lib\site-packages (from nltk) (1.3.2)
Requirement already satisfied: regex>=2021.8.3 in c:\python312\lib\site-packages (from nltk) (2023.12.25)
Requirement already satisfied: absl-py in c:\python312\lib\site-packages (from rouge_score) (2.1.0)
Requirement already satisfied: six>=1.14.0 in c:\users\gracz\appdata\roaming\python\python312\site-packages (from rouge_score) (1.16.0)
Requirement already satisfied: torch>=1.0.0 in c:\python312\lib\site-packages (from bert_score) (2.2.1)
Requirement already satisfied: transformers>=3.0.0 in c:\python312\lib\site-packages (from bert_score) (4.36.2)
Requirement already satisfied: matplotlib in c:\python312\lib\site-packages (from bert_score) (3.8.3)
Requirement already satisfied: filelock in c:\python312\lib\site-packages (from datasets>=2.0.0->evaluate) (3.13.1)
Requirement already satisfied: pyarrow>=15.0.0 in c:\python312\lib\site-packages (from datasets>=2.0.0->evaluate) (16.1.0)
Requirement already satisfied: pyarrow-hotfix in c:\python312\lib\site-packages (from datasets>=2.0.0->evaluate) (0.6)
Requirement already satisfied: aiohttp in c:\python312\lib\site-packages (from datasets>=2.0.0->evaluate) (3.9.5)
Requirement already satisfied: pyyaml>=5.1 in c:\python312\lib\site-packages (from datasets>=2.0.0->evaluate) (6.0.1)
Requirement already satisfied: typing-extensions>=3.7.4.3 in c:\python312\lib\site-packages (from huggingface-hub>=0.7.0->evaluate) (4.9.0)
Requirement already satisfied: python-dateutil>=2.8.2 in c:\users\gracz\appdata\roaming\python\python312\site-packages (from pandas->evaluate) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in c:\users\gracz\appdata\roaming\python\python312\site-packages (from pandas->evaluate) (2024.1)
Requirement already satisfied: tzdata>=2022.7 in c:\users\gracz\appdata\roaming\python\python312\site-packages (from pandas->evaluate) (2024.1)
Requirement already satisfied: charset-normalizer<4,>=2 in c:\python312\lib\site-packages (from requests>=2.19.0->evaluate) (3.3.2)
Requirement already satisfied: idna<4,>=2.5 in c:\python312\lib\site-packages (from requests>=2.19.0->evaluate) (3.6)
Requirement already satisfied: urllib3<3,>=1.21.1 in c:\python312\lib\site-packages (from requests>=2.19.0->evaluate) (2.2.1)
Requirement already satisfied: certifi>=2017.4.17 in c:\python312\lib\site-packages (from requests>=2.19.0->evaluate) (2024.2.2)
Requirement already satisfied: sympy in c:\python312\lib\site-packages (from torch>=1.0.0->bert_score) (1.12)
Requirement already satisfied: networkx in c:\python312\lib\site-packages (from torch>=1.0.0->bert_score) (3.2.1)
Requirement already satisfied: jinja2 in c:\python312\lib\site-packages (from torch>=1.0.0->bert_score) (3.1.3)
Requirement already satisfied: colorama in c:\python312\lib\site-packages (from tqdm>=4.62.1->evaluate) (0.4.6)
Requirement already satisfied: tokenizers<0.19,>=0.14 in c:\python312\lib\site-packages (from transformers>=3.0.0->bert_score) (0.15.2)
Requirement already satisfied: safetensors>=0.3.1 in c:\python312\lib\site-packages (from transformers>=3.0.0->bert_score) (0.4.2)
Requirement already satisfied: contourpy>=1.0.1 in c:\python312\lib\site-packages (from matplotlib->bert_score) (1.2.0)
Requirement already satisfied: cycler>=0.10 in c:\python312\lib\site-packages (from matplotlib->bert_score) (0.12.1)
Requirement already satisfied: fonttools>=4.22.0 in c:\python312\lib\site-packages (from matplotlib->bert_score) (4.49.0)
Requirement already satisfied: kiwisolver>=1.3.1 in c:\python312\lib\site-packages (from matplotlib->bert_score) (1.4.5)
Requirement already satisfied: pillow>=8 in c:\python312\lib\site-packages (from matplotlib->bert_score) (10.2.0)
Requirement already satisfied: pyparsing>=2.3.1 in c:\python312\lib\site-packages (from matplotlib->bert_score) (3.1.1)
Requirement already satisfied: aiosignal>=1.1.2 in c:\python312\lib\site-packages (from aiohttp->datasets>=2.0.0->evaluate) (1.3.1)
Requirement already satisfied: attrs>=17.3.0 in c:\python312\lib\site-packages (from aiohttp->datasets>=2.0.0->evaluate) (23.2.0)
Requirement already satisfied: frozenlist>=1.1.1 in c:\python312\lib\site-packages (from aiohttp->datasets>=2.0.0->evaluate) (1.4.1)
Requirement already satisfied: multidict<7.0,>=4.5 in c:\python312\lib\site-packages (from aiohttp->datasets>=2.0.0->evaluate) (6.0.5)
Requirement already satisfied: yarl<2.0,>=1.0 in c:\python312\lib\site-packages (from aiohttp->datasets>=2.0.0->evaluate) (1.9.4)
Requirement already satisfied: MarkupSafe>=2.0 in c:\python312\lib\site-packages (from jinja2->torch>=1.0.0->bert_score) (2.1.5)
Requirement already satisfied: mpmath>=0.19 in c:\python312\lib\site-packages (from sympy->torch>=1.0.0->bert_score) (1.3.0)
Downloading bert_score-0.3.13-py3-none-any.whl (61 kB)
   ---------------------------------------- 0.0/61.1 kB ? eta -:--:--
   ------------- -------------------------- 20.5/61.1 kB 217.9 kB/s eta 0:00:01
  WARNING: Failed to write executable - trying to use .deleteme logic
ERROR: Could not install packages due to an OSError: [WinError 2] The system cannot find the file specified: 'C:\\Python312\\Scripts\\bert-score.exe' -> 'C:\\Python312\\Scripts\\bert-score.exe.deleteme'
[notice] A new release of pip is available: 24.0 -> 24.1.2
[notice] To update, run: python.exe -m pip install --upgrade pip
   -------------------------- ------------- 41.0/61.1 kB 393.8 kB/s eta 0:00:01
   ---------------------------------------- 61.1/61.1 kB 466.9 kB/s eta 0:00:00
Installing collected packages: bert_score
Requirement already satisfied: nltk in c:\python312\lib\site-packages (3.8.1)
Requirement already satisfied: click in c:\python312\lib\site-packages (from nltk) (8.1.7)
Requirement already satisfied: joblib in c:\python312\lib\site-packages (from nltk) (1.3.2)
Requirement already satisfied: regex>=2021.8.3 in c:\python312\lib\site-packages (from nltk) (2023.12.25)
Requirement already satisfied: tqdm in c:\python312\lib\site-packages (from nltk) (4.66.4)
Requirement already satisfied: colorama in c:\python312\lib\site-packages (from click->nltk) (0.4.6)
[notice] A new release of pip is available: 24.0 -> 24.1.2
[notice] To update, run: python.exe -m pip install --upgrade pip
[nltk_data] Downloading package wordnet to
[nltk_data]     C:\Users\gracz\AppData\Roaming\nltk_data...
[nltk_data]   Package wordnet is already up-to-date!
[nltk_data] Downloading package punkt to
[nltk_data]     C:\Users\gracz\AppData\Roaming\nltk_data...
[nltk_data]   Package punkt is already up-to-date!
[nltk_data] Downloading package omw-1.4 to
[nltk_data]     C:\Users\gracz\AppData\Roaming\nltk_data...
[nltk_data]   Package omw-1.4 is already up-to-date!
c:\Python312\Lib\site-packages\huggingface_hub\file_download.py:1132: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
  warnings.warn(
BartConfig {
  "_name_or_path": "roequitz/trained-distilbart-abs-0807",
  "_num_labels": 3,
  "activation_dropout": 0.0,
  "activation_function": "gelu",
  "add_bias_logits": false,
  "add_final_layer_norm": false,
  "architectures": [
    "BartForConditionalGeneration"
  ],
  "attention_dropout": 0.0,
  "bos_token_id": 0,
  "classif_dropout": 0.0,
  "classifier_dropout": 0.0,
  "d_model": 1024,
  "decoder_attention_heads": 16,
  "decoder_ffn_dim": 4096,
  "decoder_layerdrop": 0.0,
  "decoder_layers": 6,
  "decoder_start_token_id": 2,
  "dropout": 0.1,
  "early_stopping": true,
  "encoder_attention_heads": 16,
  "encoder_ffn_dim": 4096,
  "encoder_layerdrop": 0.0,
  "encoder_layers": 12,
  "eos_token_id": 2,
  "eos_token_ids": [
    2
  ],
  "extra_pos_embeddings": 2,
  "forced_eos_token_id": 2,
  "gradient_checkpointing": false,
  "id2label": {
    "0": "LABEL_0",
    "1": "LABEL_1",
    "2": "LABEL_2"
  },
  "init_std": 0.02,
  "is_encoder_decoder": true,
  "label2id": {
    "LABEL_0": 0,
    "LABEL_1": 1,
    "LABEL_2": 2
  },
  "length_penalty": 0.5,
  "max_length": 80,
  "max_position_embeddings": 1024,
  "min_length": 10,
  "model_type": "bart",
  "no_repeat_ngram_size": 3,
  "normalize_before": false,
  "normalize_embedding": true,
  "num_beams": 6,
  "num_hidden_layers": 12,
  "output_past": true,
  "pad_token_id": 1,
  "prefix": " ",
  "replacing_rate": 0,
  "save_step": 58,
  "scale_embedding": false,
  "static_position_embeddings": false,
  "student_decoder_layers": null,
  "student_encoder_layers": null,
  "task_specific_params": {},
  "torch_dtype": "float32",
  "transformers_version": "4.36.2",
  "use_cache": true,
  "vocab_size": 50264
}
c:\Python312\Lib\site-packages\huggingface_hub\file_download.py:157: UserWarning: `huggingface_hub` cache-system uses symlinks by default to efficiently store duplicated files but your machine does not support them in C:\Users\gracz\.cache\huggingface\hub\models--roequitz--trained-distilbart-abs-0807. Caching files will still work but in a degraded version that might require more space on your disk. This warning can be disabled by setting the `HF_HUB_DISABLE_SYMLINKS_WARNING` environment variable. For more details, see https://huggingface.co/docs/huggingface_hub/how-to-cache#limitations.
To support symlinks on Windows, you either need to activate Developer Mode or to run Python as an administrator. In order to see activate developer mode, see this article: https://docs.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development
  warnings.warn(message)
c:\Python312\Lib\site-packages\huggingface_hub\file_download.py:1132: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
  warnings.warn(
c:\Python312\Lib\site-packages\transformers\utils\generic.py:309: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
